libknet/Makefile.am

```diff
@@ -86,7 +86,7 @@ EXTRA_libknet_la_DEPENDENCIES	= $(SYMFILE)
 libknet_la_LDFLAGS	= $(AM_LDFLAGS) \
 			  -Wl,--version-script=$(srcdir)/$(SYMFILE) \
 			  -Wl,-rpath=$(pkglibdir) \
-			  -version-number $(libversion)
+			  -version-info $(libversion)
```

Comment:
why using info vs number? what´s the benefit? I find it a lot harder to read vs number that matches soname installed by the file.

Reply:
The info components carry the real OS-independent library version information. The version number is computed differently on different systems. Look for "vinfo_number" and "Calculate the version variables" in the generated `libtool` script. The [libtool manual](https://www.gnu.org/software/libtool/manual/html_node/Link-mode.html) also says that "new projects should use the -version-info flag instead".